lib/tasks/re_encrypt_attributes.rake

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# :nocov:
+namespace :db do
+  desc 'this task performs a db read/write to re-encrypt specified attributes for the specified model'
+  task :re_encrypt_attributes, %i[model] => :environment do |_t, args|
+    model = args[:model].camelize
+    attributes = args.extras
+
+    abort 'Need to provide `model` as argument' if model.blank?
+
+    model_classname = model.to_s.camelize.singularize.constantize
+
+    model_classname.all.each do |record|
+      attributes.each do |attribute|
```

Comment:
Suggest we filter for only attributes that are encrypted

Reply:
I was originally having the attributes passed as parameters but looks like
`modelname.encrypted_attributes.keys`
just gives me all of them, so I'll switch to that.